### PR TITLE
More Lenses Fixes

### PR DESCRIPTION
**Pull request by astog**
### Bug fixes with More Lenses
1. Citizen Management area should be properly updated/removed when needed
1. Removed code from a deprecated feature
1. Linux and Mac backward compatibility
1. Adds a new setting for builder lens: Generic hexes can now be hidden.

### CODE REVIEW

File: Assets/Text/cqui_text_settings.xml [hunk 1]
```diff
@@ -156,12 +156,18 @@
     <Row Tag="LOC_CQUI_LENSES_AUTOAPPLYSCOUTLENS" Language="en_US">
       <Text>Autoapply scout lens</Text>
     </Row>
```
Comment (chaorace) [inline Assets/Text/cqui_text_settings.xml]: If you're going to be changing LOC string names in PRs, please be sure to change them in all language files

Comment (astog) [inline Assets/Text/cqui_text_settings.xml]: Shoot, you are right. Fix, incoming...

Comment (astog) [inline Assets/Text/cqui_text_settings.xml]: Those are the new string LOCs, so translations for them are not there.

Comment (chaorace) [inline Assets/Text/cqui_text_settings.xml]: Woops, sorry! A little burnt out after staring at diffs for 3 hours 😫

Comment (astog) [inline Assets/Text/cqui_text_settings.xml]: No worries!

**Pull request merged**